config/manager/manager.yaml

```
@@ -23,10 +23,10 @@ spec:
       serviceAccountName: ocs-operator
       containers:
       - command:
-        - ocs-operator
+        - entrypoint
         args:
-        - --enable-leader-election
-        - "--health-probe-bind-address=:8081"
+          - --enable-leader-election
+          - "--health-probe-bind-address=:8081"
```

Why the extra space, list itemw should align with with the outer tag
```suggestion
        - --enable-leader-election
        - "--health-probe-bind-address=:8081"
```